docs/internal/release/release-process.md

```diff
@@ -0,0 +1,37 @@
+# Release process.
+
+## Regular release.
+
+This is a regular flow, when you are just following our usual versioning and there no additional conditions.
+
+For a `next` release, run Release workflow from Github actions panel on `development` branch.
+For a stable release, do the following:
+
+1. Create branch from `development` for next minor/major as `release/v1.2.x`.
+2. Update `libs/lerna.json` version to a new minor/major version as `1.2.0`.
```

Comment:
Shouldn't be ?
```suggestion
2. Update `libs/lerna.json` version to a new minor/major version as `1.2.0-rc.1`.
```

Reply:
But it's for stable release, not for release candidate.